pkg/cache/clusterqueue.go

```diff
@@ -75,8 +75,8 @@ type ClusterQueue struct {
 	// deleted, or the resource groups are changed.
 	AllocatableResourceGeneration int64
 
-	// ResourceStats holds nominal quota and usage for the resources of the ClusterQueue, independent of the flavor.
-	ResourceStats ResourceStats
```

Comment:
It seems we can now drop type declarations for `ResourceStats` and `QuotaStats`

Reply:
True. Done